pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[project]
+name = "canvas-course-info"
+version = "3.0.0"
+requires-python = ">=3.10,<3.11"
+dependencies = [
+  "django==3.2.14",
```

Comment:
I would loosen this but keep it on the LTS version of 3.2, so something like ~=3.2.14 or >=3.2,<3.3